components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/SequenceConfig.java

```diff
@@ -187,6 +188,22 @@ public Object clone() throws CloneNotSupportedException {
         sequenceConfig.setAuthenticatedReqPathAuthenticator(this.getAuthenticatedReqPathAuthenticator());
         sequenceConfig.requestedAcr = new ArrayList<>(this.getRequestedAcr());
         sequenceConfig.setAuthenticationGraph(this.getAuthenticationGraph());
+        sequenceConfig.setOptApplicationConfig(this.getOptApplicationConfig());
         return sequenceConfig;
     }
+
+    public void setRequestedAcr(List<String> requestedAcr) {
+
+        this.requestedAcr = requestedAcr;
+    }
+
+    public OptimizedApplicationConfig getOptApplicationConfig() {
```

Comment:
Shall we have the method name like `getOptimizedApplicationConfig` as the term `opt` has several meanings and optimization isn't one of the common meanings of it.

Reply:
Check for other similar instances.

Reply:
Fixed with [`4b4997e`](https://github.com/wso2/carbon-identity-framework/pull/4446/commits/4b4997ee7009696b196bf176bbc1702e07450522)